hknweb/settings.py

```diff
@@ -75,8 +76,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
+        'OPTIONS': {
+            'read_default_file': './db.cnf',
```

Comment:
Nice, this is good. Maybe it would be better to have a directory for config files or something? Anyway, we can worry about that later, it's not like we have too many files in the root directory of the project right now anyway.